src/main/java/com/three_tech_solutions/slot_app/controllers/implementations/StudentControllerImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.three_tech_solutions.slot_app.controllers.implementations;
+
+import com.three_tech_solutions.slot_app.data.models.Student;
+import com.three_tech_solutions.slot_app.dto.StudentResponseDTO;
+import com.three_tech_solutions.slot_app.controllers.interfaces.StudentController;
+import com.three_tech_solutions.slot_app.mappers.StudentMapper;
+import com.three_tech_solutions.slot_app.dto.StudentRequestDTO;
+import com.three_tech_solutions.slot_app.services.interfaces.StudentService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class StudentControllerImpl implements StudentController {
+
+    private final StudentService studentService;
+
+    public StudentControllerImpl(StudentService studentService, StudentMapper studentMapper) {
```

Comment:
Aca estas creando un constructor que recibe como parametro el StudentMapper, pero no es un atributo de la clase StudentControllerImpl, por lo que este constructor esta mal definido, deberiamos quitarlo

```suggestion
    public StudentControllerImpl(StudentService studentService) {
```